test/jsdocUtils.js

```diff
@@ -10,6 +10,12 @@ describe('jsdocUtils', () => {
         it('returns the primary tag name', () => {
           expect(jsdocUtils.getPreferredTagName({}, 'jsdoc', 'return')).to.equal('returns');
         });
+        it('works with the constructor tag', () => {
+          expect(jsdocUtils.getPreferredTagName({}, 'jsdoc', 'constructor')).to.equal('class');
```

Comment:
Without the `src` change, `jsdocUtils.getPreferredTagName()` returns `[Function Object]` in this case.